lib/rules/no-descending-specificity/README.md

```diff
@@ -52,6 +54,46 @@ This may lead to confusion because "the two selectors will never match the same
 
 It may be possible to restructure your CSS to remove the error, otherwise it is recommended that you disable the rule for that line and leave a comment saying why the error should be ignored. Note that disabling the rule will cause additional valid errors from being reported.
 
+### `!important`
+
+The linter only checks selectors and doesn't take `!important` into account.
+
+This can lead to valid linting errors appearing to be invalid at first glance.
+
+For example the following will cause an error:
+
+<!-- prettier-ignore -->
+```css
+a:hover { top: 10px; }
+a { top: 10px; !important; }
+```
+
+This is a correct error because the `a:hover` on line 1 has a higher specificity than the `a` on line 2.
+
+This may lead to confusion because the declaration with `!important` will apply regardless of position. However, the linter only evaluates selectors, and therefore correctly reports the error about descending specificity.
+
+It may be possible to restructure your CSS to remove the error, otherwise it is recommended that you disable the rule for that line and leave a comment saying why the error should be ignored.
+
+### Different properties
+
+The linter only checks selectors and doesn't take individual properties into account.
+
+This can lead to valid linting errors appearing to be invalid at first glance.
+
+For example the following will cause an error:
+
+<!-- prettier-ignore -->
+```css
+a:hover { top: 10px; }
+a { left: 10px; }
+```
+
+This is a correct error because the `a:hover` on line 1 has a higher specificity than the `a` on line 2.
+
+This may lead to confusion because both rules contain different declarations and there isn't any conflict between either. However, the linter only evaluates selectors, and therefore correctly reports the error about descending specificity.
+
+It may be possible to restructure your CSS to remove the error, otherwise it is recommended that you disable the rule for that line and leave a comment saying why the error should be ignored.
```

Comment:
[suggest] This is not a strong opinion. "It may be possible to restructure ..." sentence is repeated in each section under "Limitations". If reducing this repetition would make sense, moving it right after Limitation might be good, e.g.

```markdown
## Limitations

This rule has the following limitations. It may be possible to restructure your CSS to ...

### DOM structure
...
```

Reply:
Good idea! Thank you

I've moved a bit more to right after `Limitations`:

```md
## Limitations

This rule has the following limitations:

- does not have access to HTML or DOM structure
- does not consider `!important`
- does not consider individual properties

This can lead to valid linting errors appearing to be invalid at first glance.

It may be possible to restructure your CSS to remove the error, otherwise it is recommended that you disable the rule for that line and leave a comment saying why the error should be ignored. Note that disabling the rule will cause additional valid errors from being reported.
```

Please let me know what you think

Reply:
Thank you, it's much better! 👍🏼